assets/helpers/internationalisation/currency.js

```diff
@@ -1,5 +1,6 @@
 // @flow
```

Comment:
I would add a couple of unit tests to this file since it is important logic to be sure that is ok.